src/targets/gpu/hiprtc/main.cpp

```diff
@@ -58,12 +73,20 @@ int main(int argc, char const* argv[])
         std::exit(0);
     }
     std::string output_name = argv[1];
-
-    auto v = migraphx::from_msgpack(read_stdin());
-    std::vector<migraphx::gpu::hiprtc_src_file> srcs;
-    migraphx::from_value(v.at("srcs"), srcs);
-    auto out = migraphx::gpu::compile_hip_src_with_hiprtc(
-        std::move(srcs), v.at("params").to<std::string>(), v.at("arch").to<std::string>());
-    if(not out.empty())
-        migraphx::write_buffer(output_name, out.front());
+    try
+    {
+        auto v = migraphx::from_msgpack(read_stdin());
+        std::vector<migraphx::gpu::hiprtc_src_file> srcs;
+        migraphx::from_value(v.at("srcs"), srcs);
+        // clang-format off
```

Comment:
Why is clang-format turned off here?

Reply:
As far as remember, I turned it off because I found the auto-formatting ugly (I think that was the reason)
```c++
        auto out =
            migraphx::gpu::compile_hip_src_with_hiprtc(std::move(srcs),
                                                       v.at("params").to_vector<std::string>(),
                                                       v.at("arch").to<std::string>());
```
I turned auto-formating on again.